docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md

```diff
@@ -0,0 +1,94 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/alerting/configure-notifications/manage-contact-points/integrations/configure-email/
+description: Configure the Email integration to connect alerts generated by Grafana Alerting
+keywords:
+  - grafana
+  - alerting
+  - email
+  - integration
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Email
+title: Configure Email for Alerting
+weight: 300
+---
+
+## Configure Email for Alerting
+
+Use the Grafana Alerting - email integration to send email notifications when your alerts are firing. An email is sent when an alert fires and when an alert gets resolved.
+
+## Before you begin
+
+When utilizing Grafana OSS, enabling email notifications requires configuring [SMTP settings][smtp-settings] in the Grafana configuration settings. In contrast, for users of Grafana Cloud, SMTP configuration is not required.
```

Comment:
Add a note here 

Note:
This section is for Grafana OSS only. For Grafana Cloud, STMP configuration is not required.